src/templates/pycontw-2020/_includes/menu.html

```diff
@@ -81,10 +81,17 @@
 		<div class="dropdown">
 			<ul class="submenu">
 				<li><a href="{{ warmup_session }}">{% trans 'Warm-Up Session' %}</a></li>
+				<!--{% comment %}-->
+				<li><a href="{{ events_keynote_url }}">{% trans 'Keynotes' %}</a></li>
+				<!--{% endcomment %}-->
+				<li><a href="{{ events_talk_list_url }}">{% trans 'Talks' %}</a></li>
+				<!--{% comment %}-->
+				<li><a href="{{ events_tutorial_list_url }}">{% trans 'Tutorials' %}</a></li>
+				<li><a href="{{ events_openspaces_url }}">{% trans 'Open Spaces' %}</a></li>
```

Comment:
What will happen if we do not have the content at this moment?

Reply:
I expect it won't break anything because we do not have the corresponding contents yet.

Reply:
Nice 🎉 Then, I'll approve this one.

Reply:
I double checked the staging site and it does not show anything (as our expectation). I will merge the pull request.

![Selection_031](https://user-images.githubusercontent.com/3217432/86079772-dc8f7680-bac3-11ea-8d1e-b6a894a80b9e.png)